.cargo/config

```diff
@@ -0,0 +1,2 @@
+[build]
```

Comment:
Can perhaps this be adjusted to have testing included, i.e. `[build.test]` has the regular target ?

Reply:
Apparently not possible yet.